src/felles-komponenter/skjema/datovelger/datovelger.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Label, SkjemaGruppe } from 'nav-frontend-skjema';
+import Datovelger, { DatovelgerProps } from 'nav-datovelger/lib/Datovelger';
+import { FieldStateInput } from '../input/utils';
+import { datePickerToISODate, dateToDatePicker, erGyldigFormattertDato, ISODateToDatePicker } from '../../../utils';
+import styles from './datovelger.module.less';
+import classNames from 'classnames';
+
+function parseInputDate(dato?: string) {
+    return erGyldigFormattertDato(dato) ? datePickerToISODate(dato) : dato;
+}
+
+interface Props {
+    touched: boolean;
+    error?: string;
+    label: string;
+    input: FieldStateInput;
+    pristine?: boolean;
+    initialValue?: string;
+}
+
+function BasicDatovelger(props: Props & DatovelgerProps) {
+    const { label, touched, error, input, pristine, initialValue, ...rest } = props;
+    const feil = error && touched ? error : undefined;
+
+    const onChange = (date?: string) => {
+        const newValue = parseInputDate(date);
+        const customEvent = {
+            target: { name: input.name, value: newValue },
+        };
+        input.onChange(customEvent as any);
```

Comment:
any ja

Reply:
Er nødt, Eventet matcher ikke hva som egentlig skal inn. og er for mye å mocke resten av objektet